scripts/update/Updater.php

```diff
@@ -213,7 +213,7 @@ public function update($initialVersion)
                 LtiMonitorParametersService::SERVICE_ID,
                 new LtiMonitorParametersService()
             );
-            $this->setVersion('6.1.0');
+            $this->setVersion('7.0.0');
```

Comment:
Should be a separate `$this->skip('6.1.0', '7.0.0');`

Reply:
Fixed